src/main/webapp/js/utils/inputs/date-picker.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDatePicker from 'react-datepicker';
 import 'react-datepicker/dist/react-datepicker.css';
-import { parseISO } from 'date-fns';
+import { parseISO, format } from 'date-fns';
 
 const DatePicker = ({
-  onChange, value, dateFormat, ...props
+  onChange, value, dateFormat, showTimeSelect, ...props
 }) => {
   const handleChange = (date) => {
-    onChange(date);
+    const val = showTimeSelect ? date.toISOString() : format(date, 'yyyy-MM-dd');
+    const valDate = typeof date === 'string' ? date : val;
```

Comment:
you should check this before you try to format the date, because it will fail if the date is a string